include/mbedtls/aes.h

```diff
@@ -287,42 +287,30 @@ int mbedtls_internal_aes_decrypt( mbedtls_aes_context *ctx,
 #define MBEDTLS_DEPRECATED
 #endif
 /**
- * \brief           Internal AES block encryption function
- *                  (Only exposed to allow overriding it,
- *                  see MBEDTLS_AES_ENCRYPT_ALT)
+ * \brief           Old AES block encryption function without return value.
  *
```

Comment:
Can you please change this comment to 'Deprecated internal AES block encryption function without return value', which is better than 'old'.